### PR TITLE
[feature] Evidence Clock and SilhouetteBowl

### LINKED ISSUE #171: Implement Evidence Item Clock

### LINKED ISSUE #172: Implement Evidence Item Silhouette Bowl

### PR DESCRIPTION
**Pull request by JahnsMichael**
### Main changes

- add new Evidence Item, Clock and SilhouetteBowl

### Tasks List

- [close #171]
- [close #172]

#### Silhouette Bowl

- same state as Thermometer
- Behavior
  - Positive (Headless): Torus active
  - Negative (Heartless): Sphere active

https://user-images.githubusercontent.com/60375650/154018513-ece863c2-c796-4ba5-ace0-2fa095d687cc.mp4

#### Clock (sound on)

- same state as Thermometer
- Behavior
  - Different AudioClip for Active/Positive/Negative, and Base is silent
  - Currently using heartbeat SFX

https://user-images.githubusercontent.com/60375650/154018561-30128b0d-b82d-46c9-b047-c5351dc69ac2.mp4

### CODE REVIEW

**Review by naufalhilmi12345 — APPROVED** [review]
Cool! LGTM

**Review by VadimBhirawa — APPROVED** [review]
Seems good!

**Review by rafiftaris — CHANGES_REQUESTED** [review]
Nicee 👍 . Only minor changes and check docs and tc list for additional comment

**Review by rafiftaris — CHANGES_REQUESTED** [review]
private variables should use _ prefix

File: Assets/Astralization/Scripts/Interactables/Item/EvidenceItem/Clock.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
+using UnityEngine;
+using System.Collections.Generic;
+
+public class Clock : EvidenceItem
```
[the remark this answers is not in the document]
Comment (JahnsMichael) [reply Assets/Astralization/Scripts/Interactables/Item/EvidenceItem/Clock.cs:8]: done

**Comment by JahnsMichael** [comment]
> Nicee +1 . Only minor changes and check docs and tc list for additional comment

done

**Comment by JahnsMichael** [comment]
> private variables should use _ prefix

done, please re-review

**Review by rafiftaris — APPROVED** [review]
siap merge 🚀

**Pull request merged**